src/dbos-executor.ts

```diff
@@ -561,18 +561,18 @@ export class DBOSExecutor implements DBOSExecutorContext {
     }
   }
 
-  async destroy() {
+  async destroy(waitForPendingWorkflows = true): Promise<void> {
```

Comment:
For client tests, I needed to be able to shut down DBOS even when there are pending workflows. So I added this parameter + a test helper to destroy the executor global instance w/o waiting